app/controllers/solutions_controller.rb

```diff
@@ -1,2 +1,12 @@
 class SolutionsController < ApplicationController
-end
+ def create
+    @question = Question.new(question_params[:id])
```

Comment:
`binding.pry`で調べると分かるのですが，`:id`ではありません。

```suggestion
    @question = Question.find(params[:question_id])
```

さらに，`@solution`を定義しなければなりません。`solution_params`を定義して利用する形式にしましょう。